llvm/test/MC/RISCV/custom_reloc.s

```diff
@@ -34,3 +34,20 @@
   nop
   # CHECK-ASM: nop
   # CHECK-OBJ: addi zero, zero, 0x0
+
+  .reloc ., R_RISCV_VENDOR,     QUALCOMM
+  .reloc ., R_RISCV_QC_ABS20_U, my_bar + 2
+  addi a1, a1, 0
+  # CHECK-ASM: [[L3:.L[^:]+]]:
```

Comment:
The `ASM:` line, shorter than the next line by 5, should be indented to align with the following lines

ditto below

Reply:
Done